Misc/NEWS.d/next/Documentation/2019-04-19-21-45-18.bpo-36277.yVzAvJ.rst

```diff
@@ -0,0 +1,3 @@
+Add document for pdb's debug and retval commands # Don't start with "- Issue
+#<n>: " or "- bpo-<n>: " or that sort of stuff.
+###########################################################################
```

Comment:
This file should just read
```
Document pdb's debug and retval commands.
```
or
```
Document pdb's debug and retval commands. Patch contributed by ... .
```
Comments should be removed.

Reply:
Thank you for the review. I've learned that I don't really need this file.

Reply:
Yes you can remove it too, thanks for making the change :)